resources/magento2/new_module_generation_dialog.properties

```diff
@@ -0,0 +1,10 @@
+must.not.be.empty.should.contain.letters.or.numbers=Must not be empty, should contain letters or numbers
```

Comment:
Let's move this phrase to `validator` properties and rest to `common`. I think they can be reused in other forms.